index.html

```diff
@@ -12,33 +12,35 @@
     <title>Portfolio</title>
```

Comment:
[OPTIONAL]
Your Project it's nice but you also need consider to have professionalism on all your projects.
Provide detailed information about your project, how to download it, how to install it and how to run that project locally all of that information it's necessary for people without technical knowledge like a **Recruiter**.
If you check on your last and in your next projects you will see a `` General Requirements `` section the main thing for this is not to miss professionalism before to send a project for review; in other wise provide a fully description about the features implemented on your pull request description things like: what do you improve, or what do you added. I hope that this information going to be helpful for you to improve your professionalism.